src/app/Components/Toast/types.ts

```diff
@@ -18,6 +23,7 @@ export interface ToastDetails {
   onPress?: (helpers: ToastOnPressHelpers) => void
   Icon?: React.FC<IconProps>
   backgroundColor?: Color
+  duration?: ToastDuration | number
```

Comment:
(non-blocking): I don't think we need to allow custom durations. Can we change this prop to:

```suggestion
  duration?: "short" | "long"
```

? Like this we wouldn't need to import an enum everywhere we use the component.

Reply:
Thanks @olerichter00. I thought about that, and decided to allow custom because short and long are strictly ToastAndroid behaviour. If this would not be confusing to devs, I'm very happy to change it.

Reply:
Done